src/algorithms.jl

```diff
@@ -5246,6 +5246,28 @@ E. Hairer, S.P. Norsett, G. Wanner, (1993) Solving Ordinary Differential Equatio
 """
 struct Nystrom4 <: OrdinaryDiffEqPartitionedAlgorithm end
 
+"""
+    FineRKN5
```

Comment:
```suggestion
    FineRKN5()
```
To clarify how the constructor looks like